src/main/java/com/launchacademy/programmingswaglist/models/User.java

```diff
@@ -43,4 +52,12 @@
   )
   private Set<Role> roles;
 
+  @ManyToOne //Many categories to one user
+  @JoinColumn(name="category_id", nullable=false)
+  private Category category;
+
+  @OneToMany (mappedBy="users")//One user to many reviews
+  private List<Review> reviews = new ArrayList<Review>();
+
```

Comment:
please remove comments prior to submitting PRs

Reply:
changes added, thank you!